.github/workflows/lint.yml

```
@@ -0,0 +1,51 @@
+        run: pnpm install
+
+      - name: Run lint
+        run: pnpm run lint
+
+  prettier:
+    name: Run Prettier
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)